kv/txn_coord_sender.go

```diff
@@ -296,15 +317,16 @@ func (tc *TxnCoordSender) sendOne(call client.Call) {
 		var txnMeta *txnMetadata
 		if txnMeta, ok = tc.txns[string(header.Txn.ID)]; !ok {
 			txnMeta = &txnMetadata{
-				txn:             *header.Txn,
+				txn:             header.Txn,
```

Comment:
That kind of thing is just problematic in tests, but this is a decent reason to do the copy, so I changed it back.